### DISCUSSION TITLE
User Interface: design update & further development

### DISCUSSION THREAD

**Discussion by uwesteffen** [litstack/next#4] [General]
Hello World!

In the past lot of interface-components were born during the development process. In the future the design department will be actively included in this process of creating new components or re-check some design-issues that came up. 

Since we are planing to switch to tailwind, I created a small design file which gives a quick impression of what litstack can look like. 

So feel free to post your questions regarding the future function or the looks of the interface. 

https://sketch.com/s/3a200603-3684-4cdd-9785-7b6e651e3f0b

**Comment by cbl** [comment]
@uwesteffen Looks awesome! 

With most applications it happens that the menus contain many entries, by the large height of the points they fast become unclear and it is hard to find what you are looking for. Is it possible that they become narrower or could we add a choice between narrower and higher menu entries?

<img width="464" alt="Bildschirmfoto 2021-06-08 um 16 01 10" src="https://user-images.githubusercontent.com/29352871/121198864-c01ec100-c872-11eb-8d24-98bfc893f6ff.png">

**Reply by uwesteffen** [reply]
@cbl the document is updated: 
I added a small version of the main menu and optimized the dropdown a little. seems like thats the smallest we can go. 

![Nav bar small](https://user-images.githubusercontent.com/71690388/121310257-77631880-c903-11eb-8511-05e7be478f61.jpg)

**Comment by jannescb** [comment]
Maybe we should consider making the Header-Nav fixed at the top always having a white background and and thus dispense the little animation. It might also be narrowed just a tiny bit…

<img width="1079" alt="Bildschirmfoto 2021-06-09 um 09 19 51" src="https://user-images.githubusercontent.com/17292622/121311038-4c2cf900-c904-11eb-869b-e02750cf93fd.png">

**Reply by uwesteffen** [reply]
updated and a first scribble of an extended top navigation. 
also a small version. 

![image](https://user-images.githubusercontent.com/71690388/121316559-be540c80-c909-11eb-8d2f-cd3af1f5b247.png)

**Comment by uwesteffen** [comment]
Quick sketch for darkmode. 

![Basic layout Darkmode](https://user-images.githubusercontent.com/71690388/122059502-ffa65980-cdec-11eb-9126-704dcc7cb797.jpg)
